recipes/mongo-cxx-driver/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.9)
```

Comment:
Why 3.9 ?

Reply:
I took it from here:
https://github.com/bincrafters/conan-mongo-cxx-driver/blob/testing/3.4.0/CMakeLists.txt

But your question is perfectly valid: it should be 3.2, I checked mongo-cxx-drvier repo, it's always 3.2.
https://github.com/mongodb/mongo-cxx-driver/search?q=cmake_minimum_required